routes/index.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const router = express.Router();
+const Twit = require('twit');
+const config = require('./config.js')
+const { Twitter } = require('../database/database.js')
+
+router.get('/', function(req, res, next) {
+  Twitter.getAllTweets().then((results) => {
+    res.render('index', {results})
+  })
+})
+
+console.log('this bot is running...');
```

Comment:
Remove `console.logs`. Also this console.log is in the middle of the app in a nowhere zone, so I'm confused as to the reason why you are having it log here?